server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -1558,6 +1569,8 @@ abstract class Host(user: String,
     info("  starting kafka")
     startKafka
 
+    Thread.sleep(60000)
+
```

Comment:
I have few questions here:
1. do we know what is the reason that the kafka and es need more time till they start up?
2. maybe we can just do it for docker and not for all our env?since 2 minutes is a lot of time

Reply:
@lielShraga I don't know the reason. Is this the result of an upgrade?